classes/form/FormField.php

```diff
@@ -181,4 +183,16 @@ public function getConstraints()
     {
         return $this->constraints;
     }
+
+    public function setAutocomplete($autocomplete)
+    {
+        $this->autocomplete = $autocomplete;
+
+        return $this;
+    }
+
+    public function getAutocomplete()
```

Comment:
@Progi1984 will tell you to type-hint the return type, I'm sure 😄 
```suggestion
    public function getAutocompleteAttribute(): string
```

Reply:
What about every other fields which are not typed at all ?

I'm not in favor of typing only on method while every siblings methods are not typed at all, that should be a complete refacto don't you think?

Reply:
I understand the will for homogeneity, but on a practical aspect, what would be the true downside of type-hinting the return type of new methods we add in legacy classes ?